src/main/java/com/trivago/fastutilconcurrentwrapper/IntFloatMap.java

```diff
@@ -8,5 +8,9 @@
 
     float put(int key, float value);
 
+    float getDefaultValue();
+
     float remove(int key);
+
+    boolean remove(int key, float value);
```

Comment:
Could you pls add a test for the method

Reply:
Thanks for looking at the PR!

I started adding tests and realized that there are actually no tests so far at all. Is there any way to automatically create the repeated classes? It is - to put it mildly - a bit tedious to recreate the exact same of tests for all different types (`Concurrent*`, `ConcurrentBusyWaiting*` and `Primitive*Wrapper`) of implementations *and* for all different key-value type combinations.

Due to this, so far I only added a test class for the `*LongLong*` variants.

If these look good, I could add more tests for the other variants, although in the longer run we should add some templating to create the classes.

Reply:
Thank you.

For quite some time I've been thinking about some generalization of every map implementation (to have a common abstract class, and hence common tests), let's see how it will go. 

For now, your proposed approach looks good.